message_private.module

```diff
@@ -371,3 +382,103 @@ function _message_private_inbox_access() {
   // Allow if the user has the bypass permission
   return AccessResult::allowedIfHasPermission($account, 'bypass private message access control');
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function message_private_form_entity_view_display_edit_form_alter(&$form, FormStateInterface $form_state) {
+  $display = $form_state->getFormObject()->getEntity();
+//  $form['fields']['author'] = [
```

Comment:
Let's not have any commented out chunk of code unless they are explicitly preceded by a `@todo` detailing what the future intention is.